pkg/registry/cluster/storage/storage.go

```diff
@@ -128,3 +128,8 @@ func (r *StatusREST) Update(ctx context.Context, name string, objInfo rest.Updat
 func (r *StatusREST) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
 	return r.store.GetResetFields()
 }
+
+// Destroy cleans up its resources on shutdown.
+// Not implemented.
+func (r *StatusREST) Destroy() {
```

Comment:
// Given that underlying store is shared with REST,
// we don't destroy it here explicitly.